example/web/mod.ts

```diff
@@ -1,7 +1,6 @@
-import deployment from "./deployment.ts";
-import service from "./service.ts";
+import webService from "./web-service.ts";
 
 export default [
-  deployment,
-  service,
+  webService.deployment,
```

Comment:
it's a hack for now because `yaml.stringify` could not separate deployment from service with '---'. When I figure out the way to do it then only the web service will be exported.

Reply:
I came up with 2 solutions on it.

1. Modify [this line](https://github.com/in-fun/kube-script/blob/main/main.ts#L17) to make it also iterate (have another for loop) for constructs.
2. Make `service` add a public method eg `manifest` that returns list of resource manifests eg `[deployment, service]` here, then `mod.ts` only need to concat it with other objects.